homeassistant/helpers/config_validation.py

```diff
@@ -1455,6 +1456,32 @@ def _base_trigger_validator(value: Any) -> Any:
     }
 )
 
+
+_SCRIPT_PARALLEL_SEQUENCE = vol.Schema(
+    {
+        **SCRIPT_ACTION_BASE_SCHEMA,
+        vol.Required(CONF_SEQUENCE): SCRIPT_SCHEMA,
+    }
+)
+
+_parallel_sequence_action = vol.All(
+    # Wrap a shorthand sequences in a parallel action
+    SCRIPT_SCHEMA,
+    lambda config: {
```

Comment:
I'm a little rusty on all this, but why wrap it in `{ "sequence": ... }` ?

Reply:
```yaml
parallel:
  - x
  - x
  - sequence:
    - x
    - x
```

becomes:

```yaml
  - sequence:
     - x  
  - sequence:
     - x  
  - sequence:
    - x
    - x
```

This keeps processing uniform and debug traces nice and clean.

Reply:
Do we want to have `seuqence` for YAML readability versus a list of lists ?

Reply:
🤷 Technically this works:

```yaml
parallel:
  - x
  - x
  - - x
    - x
```

Would not recommend it for readability (hence the "fake" sequence for that matter). The key only exists for readability.

Reply:
alright let's keep it.